src/encoded/static/components/static-pages/HomePage.js

```diff
@@ -11,6 +11,7 @@ import { requestAnimationFrame } from '@hms-dbmi-bgm/shared-portal-components/es
 
 import { navigate } from'./../util';
 import { Announcements, HomePageCarousel } from './components';
+import { pageTitleViews, PageTitleContainer, TitleAndSubtitleUnder, OnlyTitle } from './../PageTitle';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['OnlyTitle' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956)